rts/motoko-rts-tests/src/gc.rs

```diff
@@ -446,6 +452,7 @@ impl GC {
     fn run(&self, heap: &mut MotokoHeap, _round: usize) -> bool {
         let heap_base = heap.heap_base_address();
         let static_roots = Value::from_ptr(heap.static_root_array_address());
+        let mut region_0 = Value::from_scalar(0);
```

Comment:
Maybe, it could make sense to also use a region object during the test, not only 0 which means `unused`.